frontend/public/components/storage/create-pvc.tsx

```diff
@@ -271,29 +218,15 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
           )}
         </p>
       </div>
-      <label className="control-label" htmlFor="volume-mode">
-        {t('public~Volume mode')}
-      </label>
       <div className="form-group">
-        {storageProvisioner.includes(cephStorageProvisioners[1]) ? (
-          <>
-            {t('public~Filesystem')}
-            <FieldLevelHelp>
-              {t('public~Only filesystem volume mode is available for cephfs')}
-            </FieldLevelHelp>
-          </>
-        ) : (
-          getVolumeModeRadios().map((radio) => (
-            <RadioInput
-              {...radio}
-              key={radio.value}
-              onChange={handleVolumeMode}
-              inline
-              checked={radio.value === volumeMode}
-              name="volumeMode"
-            />
-          ))
-        )}
+        <VolumeModeSelector
+          onChange={setVolumeMode}
+          provisioner={storageProvisioner}
+          accessMode={accessMode}
+          className=""
```

Comment:
You should be able to remove this prop instead of setting it to an empty string. (Presumably `className` is optional.)

Reply:
ack